_pages/en/strategy-target-solution-delivery-model.md

```diff
@@ -163,27 +164,29 @@ Stakeholders are expected to still comply with existing policy instruments inclu
 
 ### Architecture
 
-1. IT Solutions use loose coupling architectures, as defined by the <_[Adopt, Build, Buy Strategy](https://sara-sabr.github.io/ITStrategy/strategy-build-vs-buy.html)_\>
-2. IT Products do not depend on other IT Products to run in production
-3. IT Products can test, deploy, and make technical changes without dependencies on other IT Product DevOps teams
-4. DevOps teams have the authority to choose their IT Products technical stacks, unless the technical stacks are in the Containment or Retirement category, as defined by the \<_[Technology Standards Management](https://sara-sabr.github.io/ITStrategy/strategy-build-vs-buy.html)_\>
-5. IT Products expose their functionalities and data via secure APIs
-6. IT-Enabled Projects affecting Legacy Systems are scoped to use the \<_strangler design pattern_\> in alignment with the \<_Target Architecture Vision for Legacy System_\>
-7. Legacy Systems are systems in the \<_ESDC Legacy Systems Standard List_\>
+1. IT Solutions are loosely coupled to Business Capabilities, as defined by the <_[Adopt, Build, Buy Strategy](https://sara-sabr.github.io/ITStrategy/strategy-build-vs-buy.html)_\>
+2. Applications, making up IT Solutions, are loosely coupled with the organization's Data
+3. Software components, making up Applications, are loosely coupled between them
+4. IT Products can test, deploy, and make technical changes without dependencies on other IT Product DevOps teams
+5. DevOps teams have the authority to choose their IT Products technical stacks, unless the technical stacks are in the Containment or Retirement category, as defined by the \<_[Technology Standards Management](https://sara-sabr.github.io/ITStrategy/strategy-build-vs-buy.html)_\>
```

Comment:
I wouldn't say new bricks. I interpret new "bricks" as an "application capability". There may be more than one option in a single brick. The idea here is to not constraint DevOps teams with pre-existing standards just because they are standards meant for the Enterprise. The Tech Brick process allows for "emerging tech" to be added. But right now the process to go at it is cumbersome and heavy.

Tech Arch reacted to this statement. But after speaking to them, they are open to the idea. Just would need to be more encadré so that it's not free for all.

The main idea behind this statement: current Technical Standard are framed as Enterprise (so anyone within ESDC). But some user profiles (i.e. dev and ops) are making tools for the rest of ESDC. They need some level of freedom to operate.